resources/scss/componets/02_typography.scss

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Rubik&display=swap');
+
+*{
+    font-family: 'Rubik', sans-serif;
+}
```

Comment:
Move this scss into **basics.scss** and remove this from this file please

Reply:
i move this code into basic.scss